fe/fe-core/src/main/java/com/starrocks/common/io/DeepCopy.java

```diff
@@ -76,4 +77,27 @@ public static boolean copy(Writable orig, Writable dest, Class c) {
         }
         return true;
     }
+
+    public static <T> T copyWithGson(Object orig, Class<T> c) {
```

Comment:
if we copy with json, why we still need the MetaContext?

Reply:
I'm not sure, I'll remove this

Reply:
RangePartitionInfo deserializeRange need MetaContext.  if can't find MetaContext,  will throw exception like this:
```
2022-06-20 17:47:34 [pool-20-thread-1] WARN  DeepCopy:86 - failed to copy object.
java.lang.NullPointerException: null
	at com.starrocks.server.GlobalStateMgr.getCurrentStateJournalVersion(GlobalStateMgr.java:667) ~[classes/:?]
	at com.starrocks.analysis.DateLiteral.readFields(DateLiteral.java:385) ~[classes/:?]
	at com.starrocks.analysis.DateLiteral.read(DateLiteral.java:408) ~[classes/:?]
	at com.starrocks.catalog.PartitionKey.readFields(PartitionKey.java:265) ~[classes/:?]
	at com.starrocks.catalog.PartitionKey.read(PartitionKey.java:278) ~[classes/:?]
	at com.starrocks.common.util.RangeUtils.readRange(RangeUtils.java:162) ~[classes/:?]
	at com.starrocks.catalog.RangePartitionInfo.deserializeRange(RangePartitionInfo.java:351) ~[classes/:?]
	at com.starrocks.catalog.RangePartitionInfo.gsonPostProcess(RangePartitionInfo.java:375) ~[classes/:?]
	at com.starrocks.persist.gson.GsonUtils$ProcessHookTypeAdapterFactory$1.read(GsonUtils.java:441) ~[classes/:?]
```